src/subdomains/supporting/payment/services/transaction.service.ts

```diff
@@ -61,4 +75,36 @@ export class TransactionService {
       },
     });
   }
+
+  private async txConfirmedMail(entity: Transaction): Promise<void> {
+    try {
+      if (entity.mailTarget?.userData.mail && !DisabledProcess(Process.TX_MAIL)) {
```

Comment:
Maybe `sendTxAssignedMail` would be a better name? Confirmed sounds like it's already finished 😉